src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -314,7 +314,7 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
       </div>
       {isLoading && <ActionSidebarLoadingSkeleton />}
       <div
-        className={clsx('flex flex-grow', {
+        className={clsx('flex flex-grow overflow-y-auto', {
```

Comment:
@rumzledz I think this is already added in your #3621 PR, so let's remove it from here 👍 or base this branch on top of the other one